src/main/java/com/neonlab/product/dtos/UserDetailsDto.java

```diff
@@ -0,0 +1,25 @@
+
+package com.neonlab.product.dtos;
+
+
+import com.neonlab.product.entities.Order;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.OneToMany;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class UserDetailsDto {
+    private String name;
+    private String email;
+    private String primaryPhoneNo;
```

Comment:
An UserDto already exists in the common. Please check if that can be used 

cc- @sjha24